files/build_templates/cbf_config.j2

```diff
@@ -0,0 +1,82 @@
+{
+    "DSCP_TO_TC_MAP": {
```

Comment:
DSCP_TO_TC_MAP is already used by qos_config.j2.

If the intention is to use the same mapping, no need to add here.  QoS config generation will create the config for DSCP_TO_TC_MAP. Having in two places will result in race condition where the last config generation will overwrite the DSCP_TO_TC_MAP.